airflow/dag/serialization/serialized_dag.py

```diff
@@ -40,18 +40,17 @@ class SerializedDAG(DAG, Serialization):
     """
     # Stringified DAGs and operators contain exactly these fields.
     # FIXME: to customize included fields and keep only necessary fields.
-    _included_fields = set(vars(DAG(dag_id='test')).keys()) - {
+    _included_fields = list(set(vars(DAG(dag_id='test')).keys()) - {
         'parent_dag',
-    }
+    })
```

Comment:
Maybe change the base type to `Iterable[str]`?

Reply:
Done